apps/site/next-data/generators/downloadSnippets.mjs

```diff
@@ -11,7 +11,7 @@ import { availableLocaleCodes } from '../../next.locales.mjs';
  * This method is used to generate the Node.js Website Download Snippets
```

Comment:
Same here, revert unrelated stylistic changes.

Reply:
this file is indeed mostly stylistic, would you prefer a separate PR for it? arrows are intended for inline callbacks, not for the main purpose of a module.